src/GridView/GridView.js

```diff
@@ -3,18 +3,25 @@ import {Container} from "react-grid-system";
 import DonutChart from "./DonutChart";
 import HierarchyStepper from "./HierarchyStepper";
 import SearchBar from "./../Utils/SearchBar";
+import FilterBar from "./../Utils/FilterBar"
 import CardGrid from "./CardGrid";
 import {connect, PromiseState} from "react-refetch";
 import ConnectionComponent from "./../Utils/ConnectionComponent.js";
 import AppStyles from "./../AppStyles";
 import config from './../config/config.js';
 import Header from "../Header";
 
+
 class GridView extends ConnectionComponent {
 	constructor() {
 		super();
 		this.getChildEntityTypes = this.getChildEntityTypes.bind(this);
 		this.getEntityType = this.getEntityType.bind(this);
+		this.handleFilterChange = this.handleFilterChange.bind(this);
```

Comment:
https://medium.freecodecamp.com/react-binding-patterns-5-approaches-for-handling-this-92c651b5af56 -> Approach #4 
1. super call
2. state definition
3. bindings